src/ethereum/frontier/eth_types.py

```diff
@@ -128,7 +136,37 @@ class Receipt:
     post_state: Root
     cumulative_gas_used: Uint
     bloom: Bloom
-    logs: List[Log]
+    logs: Tuple[Log, ...]
 
 
 State = Dict[Address, Account]
+
+
+def modify_state(
+    state: State, address: Address, f: Callable[[Account], None]
+) -> None:
+    """
+    Modify an `Account` in the `State`.
+    """
+    state[address] = modify(state[address], f)
```

Comment:
Can/should we implement initializing and removing empty accounts?

Reply:
I'd like to deal with both those problems in my upcoming state refactor rather than implementing a temporary fix now.